install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md

```diff
@@ -0,0 +1,154 @@
+# Migration Guide from Thanos to Kubecost 2.0 (Aggregator)
+
+This tutorial is intended to help our users migrate from the legacy Thanos federation architecture to [Kubecost 2.0 (Aggregator)](aggregator.md). There are a few requirements in order to successfully migrate to Kubecost 2.0. This new version of Kubecost includes a new backend Aggregator which handles the ETL data built from source metrics more efficiently. Kubecost 2.0 will provide new features, optimize the Kubecost's UI performance, and enhance the user experience.
+
+## Key changes
+* Assets and Allocations are now paginated
+   * Frontend/API users never have full set of information
+   * Use `offset`/`limit` parameters to govern pagination
+* New data available for querying every 2 hours (can be adjusted)
+* Embedded DuckDB database serves queries
+   * Data no longer queried directly from bingen files
+   * Substantial query speed improvements even when pagination not in effect
+* Data ingested into independent Aggregator component
+* Idle (sharing), Cluster Management sharing, and Network are computed a priori
+* Distributed tracing integrated into core workflows
+* No more pre-computed "agg stores"
+   * Request-level caching still in effect
+
+<details>
+
+<summary>Diagram of Aggregator:</summary>
+
+![aggregator-diagram](/images/aggregator/aggregator-diagram.png)
+
+</details>
+
+## Migration Path 
+
+<details>
+
+<summary>Diagram of Aggregator:</summary>
+
+![migration-diagram](/images/aggregator/migration-diagram.png)
+
+</details>
+
+* All steps are done on the primary except for step 8.
```

Comment:
Am I misinterpreting this? Step 8 is specifically done on the primary cluster

Reply:
Step 7 is also done on the secondary clusters

Reply:
I've went ahead and removed these two bullet points to reduce any confusion:

> * All steps are done on the primary cluster except for step 8.
> * Nothing needs to be done on the secondary clusters for the migration to be successful. The Thanos sidecar on the secondary clusters will have no impact on the migration or functionality.